packages/editor/src/browser/editor-decoration.ts

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (C) 2017 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { injectable, inject } from "inversify";
+import { Range } from "./editor";
+import { EditorManager } from "./editor-manager";
+import URI from "@theia/core/lib/common/uri";
+
+export const EditorDecorationsService = Symbol('EditorDecorationsService');
+
+export interface EditorDecorationsService {
```

Comment:
Do we need an interface? There does not seem to be other implementations of `EditorDecorationService`

Reply:
no, no need. should be just a `class EditorDecorationService`